_data/meltano/extractors/tap-zammad/lawiss.yml

```
@@ -0,0 +1,34 @@
+capabilities:
+- about
+- state
+- catalog
+- discover
```

```suggestion
- discover
- stream-maps
- schema-flattening
```

These are automatically included with the SDK!